reason-relay-documentation/docs/subscriptions.md

```diff
@@ -4,4 +4,111 @@ title: Subscriptions
 sidebar_label: Subscriptions
 ---
 
-_WIP_.
+#### Recommended background reading
+
+- [Subscriptions in Relay from the Relay documentation](https://relay.dev/docs/en/subscriptions)
+
+## Subscriptions in Relay
+
+Subscriptions use a different network configuration than _queries_ and _mutations_. They use websocket to send and receive data from the server. We will need to configure the Relay environment by providing a `subscriptionFunction` when creating the Relay network. Relay will call this function every time we want to create a new subscription. We will also need to pick a subscription client. In this example we will be using [subscription-transport-ws](https://github.com/apollographql/subscriptions-transport-ws)
+
+```reason
+/* RelayEnv.re */
+let subscriptionClient = SubscriptionTransportWS.make("wss://localhost:9090", ());
+
+let subscriptionFunction = (
+  config: ReasonRelay.operation,
+  variables: 'a,
+  _cacheConfig: ReasonRelay.cacheConfig)
+  : ReasonRelay.Observable.t => {
+    let query = config.text;
+    let subscriptionQuery = { query, variables };
+    ReasonRelay.Observable.make(sink => {
+      let observable =
+        SubscriptionTransportWS.request(subscriptionClient, subscriptionQuery);
+
+      let subscription = SubscriptionTransportWS.subscribe(observable, sink);
+      Some(SubscriptionTransportWS.unsubscribe(subscription));
```

Comment:
This is not really clear since the bindings for SubscriptionTransportWS are in my case:
```reason
type unsubscribeFn: unit => unit;
[@bs.send]
external unsubscribe : (subscription) => unsubscribeFn = "unsubscribe"
```
So this is returning the `unsubscribeFn`. It just depends on the subscription client that you use.

I can change this to be more transparent in what is going on.
```suggestion
      let unsubscribeFn: unit => unit = SubscriptionTransportWS.unsubscribe(subscription);
      Some(unsubscribeFn);
```